test/rsa_complex.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * Check to see if there is a conflict between complex.h and openssl/rsa.h.
+ * The former defines "I" as a macro and earlier versions of the latter use
+ * for function arguments.
+ */
+#if defined(__STDC_VERSION__)
+# if __STDC_VERSION__ >= 199901L
+#  include <complex.h>
```

Comment:
For DJGPP, a `<complex.h>` is available only for C++. But this creates a `rsa_complex.exe`:
```diff
--- a/test/rsa_complex.c  2018-12-21 22:13:43
+++ b/test/rsa_complex.c 2018-12-13 16:44:01
@@ -13,7 +13,7 @@
  * for function arguments.
  */
 #if defined(__STDC_VERSION__)
-# if (__STDC_VERSION__ >= 199901L) && !defined(__DJGPP__)
+# if __STDC_VERSION__ >= 199901L
 #  include <complex.h>
 # endif
 #endif
```

(too lazy to create yet another PR for this).